content/blog/2018-06-06-2018-unconf-security-recap.md

```diff
@@ -0,0 +1,53 @@
+
+Unconf projects x: 
+
+---
+slug: "unconf_recap_x"
+title: "Unconf18 projects x: middlechild, defender, ropsec, keybase"
+date: 2018-06-xx
+authors:
+  - name: Karthik Ram
+categories: blog
+topicid: 
+tags:
+- community
+- ropensci
+- meetings
+- unconf
+---
+
+As part of our series summarizing all projects from this year's unconf I'm excited to dive into all the security related offerings from this year.
+
+# `middlechild`
+
+**Summary:** This package provides a R interface to the Man-in-the-middle (MITM) proxy and allows R users to intercept, modify, and introspect network traffic. The package provides functionality to download, install, configure and launch `mitmproxy`. In addition to helping test API packages and identify dangerous calls, R users can also route their browser traffic through a specific port and capture all network traffic in R. 
+
+**Team:** [Ildiko Czeller](https://github.com/czeildi), [Karthik Ram](https://github.com/karthik), [Bob Rudis](https://github.com/hrbrmstr), [Kara Woo](https://github.com/karawoo)   
+
+**GitHub:** [https://github.com/ropenscilabs/middlechild](https://github.com/ropenscilabs/middlechild)
+
+# `defender`
```

Comment:
for pkg name headings Maëlle used `###`. I haven't previewed yours - what heading type do you recommend that others can copy?